scripts/copy_ios_push_file.js

```diff
@@ -177,4 +176,66 @@ module.exports = function(ctx) {
         });
     }
 
```

Comment:
@jcesarmobile maybe add a comment here to remove those methods once we support corodova6 as min platform